newsplease/helper_classes/url_extractor.py

```diff
@@ -19,9 +23,21 @@
 # len(".markdown") = 9
 MAX_FILE_EXTENSION_LENGTH = 9
 
+# Set of sitemap patterns
+SITEMAP_PATTERNS = [
```

Comment:
Nice PR! I guess we should move this list into the cfg files so that users can easily modify them (or disable this functionality entirely) without having to touch code.

Reply:
Thank you for your feedback ! You're absolutely right 💯 I've made the updates. This will ensure the same behavior as an option that, by default, will not impact the standard functionality